packages/back-end/src/services/stats.ts

```diff
@@ -122,7 +124,11 @@ result = analyze_metric_df(
   weights=weights,
   type=type,
   inverse=inverse,
-  # engine=StatsEngine.FREQUENTIST
+  engine=${
+    statsEngine === "bayesian"
+      ? "StatsEngine.BAYESIAN"
+      : "StatsEngine.FREQUENTIST"
```

Comment:
Can you double check and confirm that existing organizations will default to Bayesian if they don't have the setting yet?  I know you have a default parameter value for this function, but what if it's any empty string or something like that?  Might be safer to switch this logic and only use frequentist if it's explicitly set to that.

Reply:
Yep, totally agree. Fixed